templates/policy-controller.yaml

```diff
@@ -152,3 +95,36 @@ spec:
         - name: calicoctl
           hostPath:
             path: /opt/calicoctl
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: calico-node
```

Comment:
We don't need the calico-node ClusterRole or ClusterRoleBinding, since we run calico-node outside of k8s.

On the other hand, I guess it doesn't hurt to include them, and maybe it will be easier to keep our templates in sync with upstream if we just leave it in. So I think I'm good with this.

Reply:
It would also help the transition once we do run workloads on the master.